core/controllers/acl_decorators.py

```diff
@@ -2345,6 +2346,48 @@ def test_can_change_topic_publication_status(self, **kwargs):
     return test_can_change_topic_publication_status
 
 
+def can_access_classroom_page(handler):
+    """Decorator to check access to classroom page.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that now also checks if
+            one can access the classroom page.
+    """
+
+    def test_can_access(self, classroom_name, **kwargs):
```

Comment:
Ok, I can remove the check for login, but I kept it there due to the tracking of skill mastery in topics which requires logged in user.
Also, I did not use open_access, because a check needed to be there to verify if the classroom name entered is valid or not.

Reply:
Ok

Reply:
Done